.travis.yml

```diff
@@ -7,3 +7,4 @@ before_install:
 install:
 - npm install
 - bower install
+sudo: false
```

Comment:
As long as its in a separate commit. Should also enable caching, see, for example, https://github.com/jquery/qunit/pull/835